docs/src/pages/[platform]/connected-components/authenticator/configuration/signUpAttributes.all.android.mdx

```diff
@@ -0,0 +1,24 @@
+```dart{3-16}
+val authenticatorState = rememberAuthenticatorState(
+    signUpForm = {
+        username(),
+        password(),
+        confirmPassword(),
+        email(),
+        phoneNumber(),
+        birthdate(),
+        familyName(),
+        givenName(),
+        middleName(),
+        name(),
+        nickname(),
+        preferredUsername(),
+        profile(),
+        website(),
```

Comment:
There's a similar code snippet in signUpAttributes.default.android.mdx but this file has a comma after each attribute while the other does not. Should there be commas after each attribute?

Reply:
There should not be